test/runner/selenium/createDriver.js

```diff
@@ -0,0 +1,49 @@
+import { Builder, Capabilities } from "selenium-webdriver";
+import Chrome from "selenium-webdriver/chrome.js";
```

Comment:
BTW, we might want to eventually consider switching to Chrome for Testing here: https://developer.chrome.com/blog/chrome-for-testing

Reply:
The chrome driver works with chrome for testing and is the default if you don't have chrome already installed. I don't think it's necessary in GitHub actions and you can force it here by setting `CHROME_BIN`.